error: Not found `outside` declare in the `widget!`, `on` item only allow to observe widget declared in the `widget!` macro which itself located in.
  --> tests/ui/declare/on_event_observe_not_self_declare_fail.rs:10:14
   |
10 |           on outside { tap: |_| { }}
   |              ^^^^^^^

warning: assigned id but not be used in anywhere.
 --> tests/ui/declare/on_event_observe_not_self_declare_fail.rs:6:11
  |
6 |       id: outside,
  |           ^^^^^^^
  |
help: Remove this line.
 --> tests/ui/declare/on_event_observe_not_self_declare_fail.rs:6:11
  |
6 |       id: outside,
  |           ^^^^^^^
